### ISSUE TITLE
smoke_catalina_start_up_ios timeToFirstFrameMicros and timeToFirstFrameRasterizedMicros benchmarks now consistently red...

### ISSUE THREAD

**Issue by kf6gpe** [flutter/flutter#57221]
Here's timeToFirstFrameMicros recently:

![Screen Shot 2020-05-14 at 9 35 24 AM](https://user-images.githubusercontent.com/1435716/81960767-4b714880-95c6-11ea-857e-9af5ec64bb40.png)

And here's 
timeToFirstFrameRasterizedMicros

![Screen Shot 2020-05-14 at 9 35 04 AM](https://user-images.githubusercontent.com/1435716/81960703-35fc1e80-95c6-11ea-888b-6c7c00ca584b.png)

Hard to tell a culprit.

**Comment by gaaclarke** [comment]
@liyuqian  Something seems to have changed but it's not looking good trying to find the issue.  Here are my first results:


revision | date | timeToFirstFrameMicros | 2nd | 3rd
-- | -- | -- | -- | --
0889e14361830b7922df58c199fba4fbe1563e00 | 4/29/20 | 162353 | 141381 | 180271
c5b1dd0629ca9da507bdca1e82368ae640b76d60 | 5/1/20 | 191319 | 225663 | 185411
9bfe50cb8191c71a9378d0b07020209bea7d34fd | 5/6/20 | 172799 | 175012 | 124075

9bfe50cb8191c71a9378d0b07020209bea7d34fd was clearly in the red zone and showed up slower than the others.  In my testing it actually performed faster than 0889e14361830b7922df58c199fba4fbe1563e00 which was safely in the green zone.

**Comment by liyuqian** [comment]
Thanks @gaaclarke for the experiment! Then the regression might have been caused by some infra issues? I'm removing the TODAY label. We can further triage this issue once we get physically get back to our device lab.

**Comment by gaaclarke** [comment]
@liyuqian This test has a high coefficient of variance (stddev/mean).  I wonder if we should run it multiple times per commit or find some other way to reduce the noise.

**Comment by liyuqian** [comment]
Running it multiple times and taking the average is a good idea. Please feel encouraged to add an extra `repetition` parameter to [StartupTest][1], and add a for loop similar to how we currently does with a [memory test][2]. I'll happily review it :)

[1]: https://github.com/flutter/flutter/blob/1cfd792a380034e1cbd0a4c067b00165222b1f35/dev/devicelab/lib/tasks/perf_tests.dart#L220
[2]: https://github.com/flutter/flutter/blob/1cfd792a380034e1cbd0a4c067b00165222b1f35/dev/devicelab/lib/tasks/perf_tests.dart#L601

**Comment by kf6gpe** [comment]
@gaaclarke Did you get a chance to try modifying the test to run multiple times per commit? Curious about next steps here. It still shows up regularly on my morning performance sweep. Thanks!

**Comment by kf6gpe** [comment]
One more gentle ping to @gaaclarke  --- any update on whether you got to implement multiple run averaging? It looks like it got slightly better for a little while, and then started to trend badly again:
![image](https://user-images.githubusercontent.com/1435716/91771174-5fea4880-eb97-11ea-85ad-24226171de0b.png)

**Comment by gaaclarke** [comment]
@kf6gpe I never looked into it.  As roll sheriff I stopped investigating it once the test proved to be too noisy to be reliable.  I can give it a look right now if it's getting in your way (and if it is maybe p6 isn't the right priority)?

**Comment by kf6gpe** [comment]
Yeah, it probably should have been a P4 or a P5. I saw your PR, though, and it looks good! Thanks.

**Comment by kf6gpe** [comment]
Despite #64978, this is still red, so I'm leaving it open for now.
